Build issue with Webpack/Typescript/Babel

We are seeing an issue during our build when trying to use version 0.14.2 of this library, where the build errors with
**Module not found: Error: Default condition should be last one**
[Reproduced in this repo](https://github.com/joustava/reproduce-issue)

```
yarn build
yarn run v1.22.21
$ yarn build:clean && webpack --node-env=production
$ rimraf build; mkdir -p build
assets by status 149 bytes [cached] 1 asset
./src/index.ts 111 bytes [built] [code generated]

ERROR in ./src/index.ts 1:0-5:39
Module not found: Error: Default condition should be last one

webpack 5.90.3 compiled with 1 error in 165 ms
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```


A possible fix is to change the order of the 'default' key in the package.json such that it looks like:
```
"exports": {
		"import": {
			"types": "./dist/icu-message-formatter.d.ts",
			"default": "./dist/icu-message-formatter.js"
		},
		"require": {
			"types": "./dist/icu-message-formatter.d.cts",
			"default": "./dist/icu-message-formatter.cjs"
		}
	},
```

The same needs to be done in the @ultraq/function-utils repo and then (at least for us) the build is succesfull again.

--- a/package.json
+++ b/package.json
@@ -19,12 +19,12 @@
 	"types": "function-utils.d.ts",
 	"exports": {
 		"import": {
-			"default": "./function-utils.js",
-			"types": "./function-utils.d.ts"
+			"types": "./function-utils.d.ts",
+			"default": "./function-utils.js"
 		},
 		"require": {
-			"default": "./function-utils.cjs",
-			"types": "./function-utils.d.cts"
+			"types": "./function-utils.d.cts",
+			"default": "./function-utils.cjs"
 		}
 	},
 	"files": [

